### ISSUE TITLE
`bodyElement` in component definition no longer works

### ISSUE THREAD

**Issue by KeithKosh** [yapplabs/ember-tether#123]
In the most recent version of the addon (3.0.0), it seems that passing in `bodyElement` no longer works as a defined element to attach the tethered content to.

I got around it by using a default `bodyElementId` in the environment config, but the docs do list that it's still something you can define in a specific component definition. I found this code change in the most recent release:

https://github.com/yapplabs/ember-tether/pull/111/files#diff-13c6c75b06a0fd6f17074e0b83b7e3eeca42aa98b990e8eb049d0cd2cb198988R83-R94

Was it changed away from using `this.args.bodyElement` on purpose? Should we still be able to use `bodyElement` in the same way? Thanks!

**Comment by lukemelia** [comment]
@KeithKosh I don't think this was changed on purpose. The feature was previously relying on overwriting a computed property. If you or someone else reading this wants to open a PR to re-add support for passing in bodyElement, I'd happily merge it.
